components/molecule/carousel/src/CarouselContainer.js

```diff
@@ -191,7 +193,7 @@ const CarouselContainer = ({
           `${classNameBase}-arrowLeft`,
           arrowLeft ? `${classNameBase}-arrowCustom` : `${classNameBase}-prev`
         )}
-        label={arrowLeft ? null : 'Previous'}
+        label={arrowLeftLabel}
```

Comment:
is this really what we need with this `prop`? 🤔 

```suggestion
        label={arrowLeft ? null : arrowLeftLabel}
```